porcupine/plugins/text_google_search.py

```diff
@@ -0,0 +1,13 @@
+"""Search selected text on Google."""
+
+import tkinter as tk
+from porcupine import menubar, tabs
+import webbrowser
+
+def google_search(tab: tabs.FileTab) -> None:
+    selected_text = tab.textwidget.get("sel.first", "sel.last")
```

Comment:
This will error if you press Ctrl+G with no text selected. I think it should just do nothing.

Reply:
It would also be good to avoid google-searching multiple lines of text, or text that consists of space characters only. Something like `if selected_text.strip() and "\n" not in selected_text:`

Reply:
resolved in commit https://github.com/Akuli/porcupine/pull/1024/commits/b7105b4ff877b0fef23a4df394a50a879c1c701e